src/irr/asset/CBAWMeshWriter.cpp

```diff
@@ -153,6 +153,29 @@ struct LzmaMemMngmnt
 
         const ICPUMesh* mesh = static_cast<const ICPUMesh*>(_params.rootAsset);
 
+		if (!(_params.writerFlags & IAssetWriter::EWPF_MESH_IS_RIGHT_HANDED))
```

Comment:
don't flip, just save the flag into the blob (left or right handed)

Reply:
okay